src/prefabs/createForm.tsx

```diff
@@ -37,10 +38,14 @@ const beforeCreate = ({
   const [model, setModel] = React.useState(null);
   const [idProperty, setIdProperty] = React.useState(null);
   const [properties, setProperties] = React.useState([]);
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [permissions, setPermissions] = React.useState('private');
```

Comment:
which is the unused variable? maybe we can remove it.

Reply:
It is the setPermissions. While it is not used right now, this should give the indication that the permission is meant to be set by pro-coders.